src/commands/distribute/releases/edit-notes.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  AppCommand,
+  CommandResult,
+  ErrorCodes,
+  failure,
+  help,
+  success,
+  shortName,
+  longName,
+  required,
+  hasArg,
+} from "../../../util/commandline";
+import { AppCenterClient, clientRequest, models } from "../../../util/apis";
+import { out } from "../../../util/interaction";
+import { inspect } from "util";
+import { handleHttpError } from "../../../util/apis/create-client";
+import * as Pfs from "../../util/misc/promisfied-fs";
```

Comment:
`error TS2307: Cannot find module '../../util/misc/promisfied-fs'.`

Reply:
Oh...Type script. Sorry guys but that's my first time with it ;)

Reply:
That's okay. Just a missing "../" here. Hopefully we can all learn in the process. :)